app/views/games/show.html.erb

```diff
@@ -4,7 +4,12 @@
   <% (0..7).each do |y| %>
     <tr>
     <% (0..7).each do |x| %>
-      <td data-x="<%= x %>" data-y="<%= y %>"></td>
+      <td data-x="<%= x %>" data-y="<%= y %>">
+      <% piece = Piece.find_by(x_position: x, y_position: y, game_id: @game.id) %>
+      	<% if piece.present? %>
+      		<div class="piece" data-black="<%= !piece.black? %>"><%= piece.unicode.html_safe %></div>
```

Comment:
Fairly soon we'll be getting drag-and-drop, and we will probably wan to know the ID of the piece being moved - can you add `data-id="<%= piece.id %>"` here as well?